.changeset/sweet-schools-jump.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+chore: removed GitHub pages deployment
```

Comment:
That was quicker than I thought. 👏 

Where did you set `VERCEL_FORCE_NO_BUILD_CACHE`?

Reply:
Updated [comment](https://github.com/FuelLabs/fuels-ts/pull/2780#issuecomment-2236002557) with related links :)

Reply:
I did have assistance from the wizard @Dhaiwat10 🧙🏼

Reply:
Not a request, just a question: do you know if it could be set in the workflow?

Reply:
Going to assume you mean GitHub workflow.

It looks [possible](https://vercel.com/docs/cli/env) and I would favour having as much setup in GitHub as we can. Not much of a fan of the Vercel magic blackbox.